src/DataProtection/docs/Az.DataProtection.md

```diff
@@ -113,6 +116,18 @@ This operation will stop backups for backup instance
 ### [Sync-AzDataProtectionBackupInstance](Sync-AzDataProtectionBackupInstance.md)
 Sync backup instance again in case of failure\r\nThis action will retry last failed operation and will bring backup instance to valid state
 
+### [Test-AzDataProtectionBackupInstanceReadiness](Test-AzDataProtectionBackupInstanceReadiness.md)
+Validate whether adhoc backup will be successful or not
+
+### [Test-AzDataProtectionBackupInstanceRestore](Test-AzDataProtectionBackupInstanceRestore.md)
+Validates if Restore can be triggered for a DataSource
+
+### [Test-AzDataProtectionBackupVaultNameAvailability](Test-AzDataProtectionBackupVaultNameAvailability.md)
```

Comment:
remove Test-AzDataProtectionBackupVaultNameAvailability and Test-AzDataProtectionFeatureSupport